go.mod

```diff
@@ -6,7 +6,8 @@ require (
 	github.com/Masterminds/semver v1.5.0
 	github.com/Masterminds/semver/v3 v3.1.1
 	github.com/ghodss/yaml v1.0.0
-	github.com/giantswarm/apiextensions/v3 v3.13.0
+	github.com/giantswarm/apiextensions/v3 v3.13.1-0.20210104133707-4fe232b93488
```

Comment:
I think this should be reverted?

Reply:
I was testing before releasing apiextensions.